utils/analyzers/time_spent.rb

```diff
@@ -0,0 +1,66 @@
+require 'business_time'
+
+module Util
+  module Analyzers
+    class TimeSpent
```

Comment:
I'm having trouble with this class name as we've preferred action-"er" names throughout the project. How about `TimeTracker` or `TimeCounter` or something like that?